### PR TITLE
COP-12748: RoRo v1 users with access to RORO v2

### PR DESCRIPTION
**Pull request by Shaks**
RoRo v1 users with access to RORO v2 via a dropdown

### CODE REVIEW

File: client/src/pages/targets/v2/TargetsListPage.test.jsx [hunk 1]
```diff
@@ -190,17 +190,31 @@ describe('TargetsListPage', () => {
...
+    GROUP_MEMBER[GROUP_IDS.RORO]
+  },`, async () => {
+    [GROUP_MEMBER[GROUP_IDS.RORO]].forEach(async (groupMember) => {
       groupsContextValue.groupMember = groupMember;
 
       const wrapper = mount(setTabAndTaskValues(tabData));
 
       const dropdown = wrapper.find(Select);
+      expect(dropdown).toHaveLength(1);
+      expect(dropdown.find('select > option').length).toEqual(2);
```
Comment (charles474) [inline client/src/pages/targets/v2/TargetsListPage.test.jsx:203]: How does this test verify that the two options are roro?

File: client/src/pages/targets/v2/TargetsListPage.test.jsx [hunk 2]
```diff
@@ -190,17 +190,31 @@ describe('TargetsListPage', () => {
...
+    GROUP_MEMBER[GROUP_IDS.AIRPAX]
+  },`, async () => {
+    [GROUP_MEMBER.BOTH].forEach(async (groupMember) => {
+      groupsContextValue.groupMember = groupMember;
+
+      const wrapper = mount(setTabAndTaskValues(tabData));
+
+      const dropdown = wrapper.find(Select);
+      expect(dropdown).toHaveLength(1);
+      expect(dropdown.find('select > option').length).toEqual(3);
```
Comment (charles474) [inline client/src/pages/targets/v2/TargetsListPage.test.jsx:217]: How does this test verify that the two options are airpax & roro?

File: client/src/pages/targets/v2/TargetsListPage.test.jsx (hunk 1 above)
Comment (Shaks) [reply client/src/pages/targets/v2/TargetsListPage.test.jsx:203]: Added a test to verify the option

File: client/src/pages/targets/v2/TargetsListPage.test.jsx (hunk 2 above)
Comment (Shaks) [reply client/src/pages/targets/v2/TargetsListPage.test.jsx:217]: Added a test to verify the options

File: client/src/pages/targets/v2/TargetsListPage.test.jsx (hunk 1 above)
Comment (michaelruocco) [reply client/src/pages/targets/v2/TargetsListPage.test.jsx:203]: I think this one would be better to be similar to how you have it below? where it is verifying the text in the options too as it is more readable that way?

**Comment by charles474** [comment]
Closing as the changes in this PR is now in the bluetargeting-dev branch.

**Pull request closed without merging**